scripts/gen_word_error_correction.mjs

```diff
@@ -0,0 +1,332 @@
+ * - Creates warnings for ambiguous terms like "listopad" (Oct in Croatian vs Nov in Czech/Polish)
+ * - Uses native JavaScript Intl APIs instead of external dependencies
+ * - Maintains backward compatibility with existing YAML structure
+ * 
+ * Input:
+ * - src/locales/word_error_correction_manual.yaml (manual corrections)
+ * - Intl.DateTimeFormat API (automatic corrections for 141+ locales)
+ * - Intl.DisplayNames API (dynamic language names)
+ * 
+ * Output:
+ * - src/locales/word_error_correction.yaml (combined corrections + ambiguous words warnings)
+ * 
+ * The generated file contains:
+ * - All manual corrections (preserved as-is)
+ * - 2400+ automatic corrections for weekdays/months
+ * - Separate "Ambiguous words" section with warning messages
+ * - Complete source documentation for transparency
+ */
+
+/*
+ * SPDX-FileCopyrightText: © 2025 Kristjan ESPERANTO <https://github.com/KristjanESPERANTO>
+ *
+ * SPDX-License-Identifier: LGPL-3.0-only
+ */
+
+import fs from 'node:fs';
+import yaml from 'yaml';
+import path from 'node:path';
+
+console.log('═══════════════════════════════════════');
+console.log('Word Error Correction Generator');
+console.log('═══════════════════════════════════════\n');
+
+// Define file paths
+const basePath = path.resolve(process.cwd(), 'src/locales');
+const manualYamlPath = path.join(basePath, 'word_error_correction_manual.yaml');
+const outputPath = path.join(basePath, 'word_error_correction.yaml');
+
+// 1. Load manual corrections
+console.log('► Loading manual corrections...');
+let manualCorrections = {};
+if (fs.existsSync(manualYamlPath)) {
+    const manualYamlContent = fs.readFileSync(manualYamlPath, 'utf8');
+    manualCorrections = yaml.parse(manualYamlContent) || {};
+    console.log(`  ✓ Loaded from: ${path.relative(process.cwd(), manualYamlPath)}`);
+    
+    // Calculate and display statistics for manual corrections
+    let totalManual = 0;
+    Object.keys(manualCorrections).forEach(category => {
+        const count = Object.keys(manualCorrections[category]).length;
+        totalManual += count;
+        console.log(`    ▪ ${category}: ${count} entries`);
+    });
+    console.log(`  → Total manual entries: ${totalManual}`);
+} else {
+    console.log(`  ✗ Manual corrections file not found: ${manualYamlPath}`);
+}
+
+// 2. Add automatic corrections for all available languages
+console.log('\n► Discovering supported locales...');
+
+// Dynamic discovery of all available locales using pattern-based testing
+console.log('  → Testing locale patterns (aa-zz)...');
+const discoveredLocales = [];
+
+// Test all 2-letter locale codes (aa to zz) for Intl support
+const letters = 'abcdefghijklmnopqrstuvwxyz';
+for (let i = 0; i < letters.length; i++) {
+    for (let j = 0; j < letters.length; j++) {
+        const locale = letters[i] + letters[j];
+        try {
+            const supported = Intl.DateTimeFormat.supportedLocalesOf([locale]);
```

Comment:
https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry

has some three letter languages.

Reply:
Wow, I didn't know that. I added it with the last commit. That brought us another 98 languages :rocket: :smiley: